packages/ldc/ldc-tools-utils-bugs.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 336bbdbc..4b9e8c88 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -736,12 +737,12 @@ endif()
+ #
+ # Auxiliary build and test utils.
+ #
+-add_subdirectory(utils)
++#add_subdirectory(utils)
```

Comment:
The CMake config in this directory would complain about not being able to find `Intrinsics.td`, despite it working fine normally and it being passed the right directory.  Rather than digging further, I just removed it, as it's never built.